tools/generator_python3.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/env python
+# coding=ascii
+###############################################################################
+# Copyright 2015-2016 University of Florida. All rights reserved.
+# This file is part of UF CTS-IT's NACCulator project.
+# Use of this source code is governed by the license found in the LICENSE file.
+###############################################################################
+
+import csv
+import os
+import re
+import sys
+# The output is too big for the command terminal, so I'm going to 
+# print everything to 'output.txt' located in the same directory.
+sys.stdout = open('output.txt', 'wt')
```

Comment:
What do you mean output is too big for the terminal?

Anyway, don't do this as it changes `stdout` for the entire program and breaks the documented usage and expected behavior.
```suggestion
```

Reply:
i meant that the terminal's history cuts off the top portion of the generator's output in vscode, so i can't copy the full output after running the generator. i will get rid of the line.